tests/redirects/map_globalconf.py

```diff
@@ -1258,9 +1258,6 @@
     url_test('/foundation/leadership-network/', 'https://foundation.mozilla.org/'),
     url_test('/foundation/advocacy/', 'https://foundation.mozilla.org/'),
 
-    # Issue 6979
-    url_test('/firefoxfightsforyou/', '/firefox/fights-for-you/'),
```

Comment:
Please update this test rather than delete it. 

Adding an additional test for the page redirect you have yet to include (`/firefox/fights-for-you/`) would also be great. Thanks.